modules/release-notes/pages/index.adoc

```diff
@@ -169,6 +169,24 @@ See xref:3.10@tigergraph-server:user-access:rbac-row-policy/row-policy-privilege
 
 == Fixed issues
 
+=== Fixed and Improved in 3.10.4
+
+==== Functionality
+
+* Fixed an issue where the `/codecheck` endpoint returned an empty response for queries with syntax or semantic errors, which previously allowed incorrect queries to be installed (GLE-10432).
```

Comment:
This one confused me: it sounds like 2 separate problems (empty response, bad queries being installed).
I finally figured out that /codecheck is an *internal endpoint* and that we use it to decide which queries to install. 
That detail was not clear.

Please improve by either (1) giving LESS detail: don't mention /codecheck.  Describe only what users see, or (2) giving MORE detail. Explain that /codecheck is internal and how it is used.